marimo/_plugins/ui/_impl/tables/default_table.py

```diff
@@ -183,7 +216,12 @@ def get_summary(self, column: str) -> ColumnSummary:
     def get_num_rows(self, force: bool = True) -> int:
         del force
         if isinstance(self.data, dict):
-            return len(next(iter(self.data.values()), []))
+            if self.is_column_oriented:
+                first = next(iter(self.data.values()), None)
+                assert isinstance(first, list)
```

Comment:
I think it can be a tuple, no? I'd remove this assert in general, don't want to crash the kernel if can help it